docs/modules/ROOT/pages/authorization.adoc

```diff
@@ -0,0 +1,90 @@
+= Authorization Methods to Access Cloud Storage
+
+== Using Secrets
+
+You can create a secret to access cloud provider resources. You can find the secret creation examples for different cloud providers in the next sections:
+
+=== Accessing Resources on AWS
+
+[source,shell]
+----
+kubectl create secret generic <secret-name> \
+--from-literal=region=<region> \
+--from-literal=access-key-id=<access-key-id> \
+--from-literal=secret-access-key=<secret-access-key>
+----
+
+See https://docs.aws.amazon.com/sdk-for-go/api/aws/session/[AWS Session] to learn about authentication procedure.
+
+=== Accessing Resources on GCP
+
+[source,bash]
+----
+kubectl create secret generic <secret-name> \
+	--from-file=google-credentials-path=<service_account_json_file>
+----
+
+See https://cloud.google.com/docs/authentication/production/[Application Default Credentials] to learn about authentication procedure.
+
+=== Accessing Resources on Azure
+
+[source,bash]
+----
+kubectl create secret generic <secret-name> \
+	--from-literal=storage-account=<storage-account> \
+	--from-literal=storage-key=<storage-key>
+----
+
+See https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal/[Azure Storage Account Keys] to learn about authentication procedure.
+
+== Using Service Accounts
+
+You can use `Service Accounts` to access cloud provider resources without providing `secretName` in `HotBackup`, `JetJob`, `UCN` custom resources. To be able to use it you need to provide `serviceAccountName` in your Hazelcast CR.
+
+.Example of Service Account Configuration
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/service-account.yaml[]
+----
+
+=== Accessing GCP Resources using Workload Identity
+Google Kubernetes Engine (GKE) Workload Identity is a feature that allows you to map a Kubernetes Service Account to a Google Cloud IAM (Identity and Access Management) Service Account so that users can manage pods permissions using IAM.
+
+Create a Service Account using the following command:
+
+[source,shell]
+----
+kubectl create serviceaccount myServiceAccount --namespace <NAMESPACE>
+----
+
+To be able to use it, you need to annotate the service account you created with the following command:
+
+[source,shell]
+----
+kubectl annotate serviceaccount myServiceAccount \
+--namespace <NAMESPACE> \
+iam.gke.io/gcp-service-account=<GCP_SA_NAME>@<GCP_PROJECT>.iam.gserviceaccount.com
+----
+
+See https://cloud.google.com/docs/authentication#service-accounts[Service Accounts] to learn more about it.
+
+=== Accessing AWS Resources using IAM Roles for Service Accounts
+IAM roles for service accounts is a feature that allows you to map a Kubernetes Service Account to a AWS IAM Role so that users can manage pods permissions using IAM.
```

Comment:
```suggestion
IAM roles for service accounts is a feature that allows you to map a Kubernetes Service Account to an AWS IAM Role so that users can manage pods permissions using IAM.
```